sound/soc/sof/sof-priv.h

```diff
@@ -398,7 +399,7 @@ struct snd_sof_dev {
 
 	/* FW configuration */
 	struct sof_ipc_dma_buffer_data *info_buffer;
-	struct sof_ipc_window *info_window;
+	struct sof_ipc_window *info_window; /* __deprecated */
```

Comment:
it'd be better if you marked 'unused after ABI X.y.z

Reply:
outdated